lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -531,6 +531,14 @@ module.exports = class MethodMappings {
           };
         },
 
+        async pageSource(options) {
+          const value = await this.driver.pageSource(options);
```

Comment:
I think according to https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_ThenableWebDriver.html#getPageSource,it should be `getPageSource`I guess

Reply:
fixed